### DISCUSSION TITLE
Ignoring the game applications which cannot be focused properly

### DISCUSSION THREAD

**Discussion by jborbik** [rxhanson/Rectangle#605] [Q&A] [answered]
Is there any way to ignore the apps if the focus prevents me from the possibility to click on the Rectangle menu bar icon? For example I wanted to ignore League of Legends but in windowed more whenever I want to give the focus to the game so that Rectangle recognizes it, I am no longer able to press anywhere outside the window. I need to switch to another app with CMD+Tab but then it is not the focused icon anymore and I have only the option to ignore the currently selected application.

Is it possible to add the app by identifier manually in some preferences file?

**Comment by rxhanson** [comment, marked as answer]
The easiest way in my opinion is to modify the settings using prefs editor (free):
https://apps.tempel.org/PrefsEditor/

Add the bundle id for whichever app/game you want to ignore in the "disabledApps" pref. If you don't have any ignored apps, it's easiest just to ignore anything so the pref is written, then you can change it when you determine the bundle id for the app/game you want to ignore. 

You can use Terminal commands for this, but it's just easier with prefs editor.

**Reply by jborbik** [reply]
Thank you!

**Reply by rxhanson** [reply]
Np. I forgot to note that you need to restart Rectangle after modifying any of its prefs from outside of it.

**Comment by jborbik** [comment]
Thank you for moving the issue to the discussion!
